js/src/jsapi.cpp

```diff
@@ -5069,11 +5080,14 @@ JS_ReportTaintSink(JSContext* cx, JS::HandleString str, const char* sink, JS::Ha
     arguments[2].setUndefined();
   }
 
-  RootedValue rval(cx);
-  JS_CallFunction(cx, nullptr, report, arguments, &rval);
+  RootedValue retVal(cx);
+  JS_CallFunction(cx, nullptr, report, arguments, &retVal);
   MOZ_ASSERT(!cx->isExceptionPending());
 
-  MaybeSpewStringTaint(cx, str);
+// Enable this with ac_add_options --enable-taintspew
+#if defined(JS_TAINTSPEW)
+  WriteTaintToFile(cx, str, retVal);
+#endif
 }
```

Comment:
Need to add a call to MaybeSpewStringTaint here in the case that JitSpew is enabled.